### PR TITLE
Open Source Readyness

### PR DESCRIPTION
**Pull request by coot**
- Removed not used cabal file
- Updated README.md file
- Added CODE_OF_CONDUCT.md file
- Added Roles & Responsibilities section to CONTRIBUTING.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1 @@
+See https://github.com/input-output-hk/cardano-engineering-handbook/blob/main/CODE-OF-CONDUCT.md
```
Comment (kevinhammond) [inline CODE_OF_CONDUCT.md:1]: Be aware that this URL might need to be updated in future, of course

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -123,3 +123,20 @@ of the content. There are two relatively straightforward ways to do this:
...
+
+We maintain [CODEOWNERS file][CODEOWNERS] which provides information who should
+review your code if it touches given projects.  Note that you need to
+acquire reviews from all code owners (even though github doesn't give a way to
+enforce it).
+
+For general architectural overview of the network code contact either:
+@coot or @dcoutts.
+
+For genereal architecutral overview of the consensus code contact either:
```
Comment (kevinhammond) [inline CONTRIBUTING.md:139]: architecutral -> architectural

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -123,3 +123,20 @@ of the content. There are two relatively straightforward ways to do this:
 
 1. The TOFU approach: put in the wrong hash and then Nix will tell you the correct one, which you can copy in.
 2. Calculate the hash with `nix-shell -p nix-prefetch-git --run 'nix-prefetch-git <URL> <COMMIT_HASH>'`
+
+## Roles and Responsibilities
+
+Maintainers of each package are listed in the corresponding `*.cabal` file.
+
+We maintain [CODEOWNERS file][CODEOWNERS] which provides information who should
```
Comment (kevinhammond) [inline CONTRIBUTING.md:131]: maintain -> maintain a

File: README.md [hunk 4]
```diff
@@ -1,16 +1,28 @@
...
-* `typed-protocols` - session type framework with support of [protocol pipelining](https://en.wikipedia.org/wiki/Protocol_pipelining)
-  * See 45min Haskell eXchange 2019 [talk](https://skillsmatter.com/skillscasts/14633-45-minute-talk-by-duncan-coutts) by @dcoutts.
-  * See three 50min Monadic Party 2019 workshop talks by @coot: [Part 1](https://www.youtube.com/watch?v=j8gza2L61nM), [Part 2](https://www.youtube.com/watch?v=oV6KSl1srL8), [Part 3](https://www.youtube.com/watch?v=nOIQCRPwmPA).
-* `ouroboros-network`- ouroboros network package which implements protocols
-  which to run ouroboros family of protocols, multiplexing layer.
-* The [`byron-proxy`](https://github.com/input-output-hk/cardano-byron-proxy) is a network protocol proxy between Byron and Shelley.
-  It now lives in a separate repository.
+* `network-mux` - implementation of a general network muliplexer.
+* `ouroboros-network-api` - shared API between `network` and `consensus` components.
+* `ouroboros-network-framework` - low level network components, e.g. snockets,
```
Comment (kevinhammond) [inline README.md:9]: snockets or sockets?

Comment (coot) [reply README.md:9]: `Snocket` combines `Socket` API & Windows Named Pipes API.

**Review by kevinhammond — APPROVED** [review]
Just spotted a few small typos.  Maybe also add

Purpose: This repository contains the core components of the network & consensus code for  the Cardano node.  It is a dependency when building the node from the cardano-node repository.

to README.md - then we can cross off another requirement

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -123,3 +123,20 @@ of the content. There are two relatively straightforward ways to do this:
...
 1. The TOFU approach: put in the wrong hash and then Nix will tell you the correct one, which you can copy in.
 2. Calculate the hash with `nix-shell -p nix-prefetch-git --run 'nix-prefetch-git <URL> <COMMIT_HASH>'`
+
+## Roles and Responsibilities
+
+Maintainers of each package are listed in the corresponding `*.cabal` file.
+
+We maintain a [CODEOWNERS file][CODEOWNERS] which provides information who should
+review your code if it touches given projects.  Note that you need to
+acquire reviews from all code owners (even though github doesn't give a way to
```
Comment (dnadales) [inline CONTRIBUTING.md:133]:
```suggestion
get approvals from all code owners (even though GitHub doesn't give a way to
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -123,3 +123,20 @@ of the content. There are two relatively straightforward ways to do this:
...
+enforce it).
+
+For general architectural overview of the network code contact either:
+@coot or @dcoutts.
+
+For general architectural overview of the consensus code contact either:
+@dnadales or @nfrisby.
```
Comment (dnadales) [inline CONTRIBUTING.md:140]: I will be simply relaying questions to others.

Comment (kevinhammond) [reply CONTRIBUTING.md:140]: That's OK - we just need to identify the maintainers as a point of principle - these can be changed later

File: CONTRIBUTING.md (hunk 5 above)
Comment (coot) [reply CONTRIBUTING.md:133]: Fixed.

**Pull request merged**